app/src/main/res/values-de/strings.xml

```diff
@@ -6,7 +6,7 @@
     <string name="action_settings">Einstellungen</string>
     <string name="action_logout">Abmelden</string>
     <string name="action_search">Suchen</string>
-    <string name="action_updates">Aktualisierungen</string>
+    <string name="action_updates">Updates</string>
```

Comment:
I prefer no anglicisms if there is already a german word for it. But I guess that's a matter of taste :)